app/views/overview/_service-instance-row.html

```diff
@@ -0,0 +1,133 @@
+<div class="list-pf-item" ng-class="{ active: row.expanded }">
+  <div class="list-pf-container" ng-click="row.toggleExpand($event)">
+
+    <div class="list-pf-chevron">
+      <div ng-include src=" 'views/overview/_list-row-chevron.html' " class="list-pf-content"></div>
+    </div>
+
+    <div class="list-pf-content">
+      <div class="list-pf-name">
+        <h3>
+          <a ng-href="{{row.apiObject | navigateResourceURL}}">
+            <span ng-bind-html="row.displayName | highlightKeywords : row.state.filterKeywords"></span>
+          </a>
+          <div class="component-label" style="padding-top: 4px;">
+            {{row.apiObject.metadata.name}}
+          </div>
+        </h3>
+        <div class="status-icons">
+          <notification-icon ng-if="!row.expanded" alerts="row.notifications"></notification-icon>
+        </div>
+      </div>
+    </div>
+
+    <div
+      class="list-pf-content hidden-xs hidden-sm">
+      <div style="width: 300px">
```

Comment:
we shouldn't be setting a fixed width here, you should be able to use bootstrap row / col classes if needed, that was the point of using the new list-pf